### ISSUE TITLE
Discriminatory Hiring Practices

### ISSUE THREAD

**Issue by mike-bailey** [wwphp-fb/conduct#13]
We have disallowed them but don't actually have anything on record for them as far as I can tell. We can't cite law because it's not the case in all countries. We could theoretically cite FB's anti-discriminatory rules but that'd be hard to make people understand and it'd be simpler to just have a CoC rule. I'll be pushing a proposal for this shortly.

Facebook's hate speech policy.
```
Hate Speech

Facebook removes hate speech, which includes content that directly attacks people based on their: 

Race,
Ethnicity,
National origin,
Religious affiliation,
Sexual orientation,
Sex, gender, or gender identity, or
Serious disabilities or diseases.
```

This also doesn't cover [ageism](https://en.wikipedia.org/wiki/Ageism) in the workforce.

**Comment by mike-bailey** [comment]
To clarify, we do have a rule regarding discrimination, but some members don't seem to realize that extends to job opportunities.

**Comment by Maikuolan** [comment]
+1.

**Comment by petk** [comment]
Checked, discussed, approved and merged via 68f15552056fb0df7607e4de6521665a1e427809
